sentry_sdk/api.py

```diff
@@ -35,16 +36,20 @@ def capture_event(event, hint=None):
 
 @hubmethod
 def capture_message(message, level=None):
+    # type: (str, Optional[Any]) -> Optional[str]
     hub = Hub.current
     if hub is not None:
         return hub.capture_message(message, level)
+    return None
```

Comment:
is this `return None` mess really necessary?

Reply:
https://github.com/python/mypy/issues/4123